backend/src/main/java/moaon/backend/event/EsEventPoller.java

```diff
@@ -0,0 +1,174 @@
+package moaon.backend.event;
+
+import co.elastic.clients.elasticsearch.core.BulkResponse;
+import co.elastic.clients.elasticsearch.core.bulk.BulkResponseItem;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import moaon.backend.global.exception.custom.CustomException;
+import moaon.backend.global.exception.custom.ErrorCode;
+import org.springframework.transaction.annotation.Transactional;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import moaon.backend.event.domain.EsEventOutbox;
+import moaon.backend.event.domain.EventAction;
+import moaon.backend.event.dto.PreparedEvent;
+import moaon.backend.event.repository.EsEventOutboxRepository;
+import moaon.backend.global.elastic.IndexNameResolver;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class EsEventPoller {
+
+    private final EsEventOutboxRepository outboxRepository;
+    private final ElasticSearchProcessor elasticSearchProcessor;
+    private final ObjectMapper objectMapper;
+    private final IndexNameResolver indexNameResolver;
+
+    private static final int BATCH_SIZE = 100;
+    private static final int MAX_RETRIES = 5;
+
+    @Scheduled(fixedDelay = 2000)
+    @Transactional
+    public void pollAndProcessEvents() {
+        List<EsEventOutbox> events = outboxRepository.findPendingEvents(BATCH_SIZE);
+        if (events.isEmpty()) {
+            return;
+        }
+
+        List<PreparedEvent> preparedEvents = new ArrayList<>();
+        List<EsEventOutbox> esFailedEvents = new ArrayList<>();
+
+        validateAndSeparateEvents(events, preparedEvents, esFailedEvents);
+
+        List<EsEventOutbox> successfulEvents = new ArrayList<>();
+
+        try {
+            if (!preparedEvents.isEmpty()) {
+                BulkResponse esResponse = elasticSearchProcessor.processEvents(preparedEvents);
+                processBulkResponse(esResponse, preparedEvents, successfulEvents, esFailedEvents);
+            }
+        } catch (IOException e) {
+            log.error("ES Bulk API 통신 실패");
+            throw new CustomException(ErrorCode.ES_PROCESSING_FAILED);
```

Comment:
```suggestion
            log.error("ES Bulk API 통신 실패", e);
            return;
```
로그만 남기고, throw없이 삼키는 게 좋을 것 같아요.

Reply:
e763348